lib/Conversion/TorchToLinalg/DataMovement.cpp

```diff
@@ -330,19 +330,27 @@ class ConvertAtenViewOp : public OpConversionPattern<AtenViewOp> {
     if (resultRank == 0)
       return rewriter.notifyMatchFailure(op,
                                          "result shape of rank 0 is invalid");
+    
+    auto sizeType = op.getSize().getType();
+    Value sizeTensor = rewriter.create<tensor::FromElementsOp>(loc, RankedTensorType::get({}, sizeType), op.getSize())
+              .getResult();
 
     // TODO: add support for case inputRank 0 expanded to size 1
     if (inputRank == 0)
-      return rewriter.notifyMatchFailure(
-          op, "unimplemented: input rank 0 is not supported");
+      // return rewriter.notifyMatchFailure(
+      //     op, "unimplemented: input rank 0 is not supported");
+      rewriter.replaceOpWithNewOp<tensor::ReshapeOp>(op, resultType, input, sizeTensor);
+      return success();
```

Comment:
The tests are failing because this return is not part of the `if` above (`if` needs braces)

Reply:
Thank you! (didn't catch that silly mistake)